snare/utils/snare_helper.py

```diff
@@ -0,0 +1,85 @@
+from distutils.version import StrictVersion
+from bs4 import BeautifulSoup
+import os
+
+
+class SnareHelper:
```

Comment:
this class contains a lot of different functions which is not connected. Why you need a class?